IS7024/Pages/HollywoodBowl.cshtml.cs

```diff
@@ -23,8 +23,8 @@ public void OnGet()
                 string sunJSON = webClient.DownloadString("https://api.sunrise-sunset.org/json?lat=34.1122&lng=118.3391&date=2020-11-02");
                 var sun = QuickTypeSun.Sun.FromJson(sunJSON);
                 var results = sun.Results.Sunset;
-                string s = results;
-                string[] vars = s.Split(':');
+                string resultsString = results;
+                string[] vars = resultsString.Split(':');
```

Comment:
Changed "s" to "resultsString" for added clarification of the variable's use

Reply:
"resutsString" is more descriptive than "s".  I'd go a few steps further:
1) Don't include the data type in a variable name.  Thus, it's now "results"
2) Results from what?  It looks like sunsets.  So I'd call it "sunsets".